test/common/shared.spells.test.js

```diff
@@ -0,0 +1,128 @@
+/* eslint-disable camelcase */
+let shared = require('../../common/script/index.js');
+
+let newUser = (addTasks = true) => {
+  let buffs;
+  let user;
+
+  buffs = {
+    per: 0,
+    int: 0,
+    con: 0,
+    str: 0,
+    stealth: 0,
+    streaks: false,
+  };
+  user = {
```

Comment:
There's a lot of data in here not actually used in the test. It doesn't need to be a complete user object, just enough to  get through the code paths being tested.